app/client/components/creditTransfer/singleCreditTransfer.jsx

```diff
@@ -60,6 +60,10 @@ class SingleCreditTransfer extends React.Component {
       const transferStatus = creditTransfer.transfer_status;
       const transferMode = creditTransfer.transfer_mode;
       const resolutionMessage = creditTransfer.resolution_message;
+      const approvers = creditTransfer.approvers || [];
```

Comment:
It could make sense to also show who rejected the transfer if that data is easily accessible